### PR TITLE
Adding new row to Managed cluster networking requirements

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25378#event-7379403139

### CODE REVIEW

File: install/requirements.adoc [hunk 1]
```diff
@@ -123,6 +123,10 @@ For the managed cluster networking requirements, see the following table:
 | The managed cluster to the channel source, which includes GitHub, Object Store, and Helm repository. This is only required when you are using application lifecycle to connect to these sources.
 | 443
 
+| Outbound to the hub cluster
+| HTTPS
+| For cluster-proxy addon on the managed cluster to register.
```
Comment (swopebe) [inline install/requirements.adoc]: add-on I believe is what that should be.

**Review by swopebe — APPROVED** [review]
approved with one change

File: install/requirements.adoc (hunk 1 above)
Comment (dockerymick) [reply install/requirements.adoc]: great catch, thanks!

**Review by swopebe — APPROVED** [review]

**Comment by dockerymick** [comment]
thanks for the reviews and approvals! @swopebe @xuezhaojun Merging this PR

**Pull request merged**